### PR TITLE
[249] Create address workload issues page

### PR DESCRIPTION
**Pull request by AbigailMcP**
## Changes proposed in this pull request

Creates the 'Address workload issues' page which lives on `/workload-reduction-toolkit/address-workload-issues`.

## Guidance to review

- Head to the homepage and click on Workload reduction toolkit
- Hover and click on the 'Address workload issues' card and check styles of link
- Compare 'Address workload issues' to prototype
- Resize the browser and check on smaller and wider widths
- Click the links and check hover/active styling (they don't link anywhere yet)

Screenshot 2024-01-12 at 13 44 43

**Hover:**

Screenshot 2024-01-12 at 13 44 41

**Focus:**

Screenshot 2024-01-12 at 13 45 02

**Focused styling for cards on Workload reduction toolkit page:**

Screenshot 2024-01-16 at 15 04 24

## Link to Trello card

https://trello.com/c/qvmAS4Lz/249-create-address-workload-issues-page

### CODE REVIEW

**Comment by mark-pkgflow** [comment]
@AbigailMcP it seems weird that we are only seeing it now, but from the build failure it says to run `bundle lock --add-platform x86_64-linux` and push again.

**Comment by mark-pkgflow** [comment]
> @AbigailMcP it seems weird that we are only seeing it now, but from the build failure it says to run `bundle lock --add-platform x86_64-linux` and push again.

Looks like we lost some platforms:

image

File: Gemfile.lock [hunk 1]
```diff
@@ -177,9 +177,7 @@ GEM
     zeitwerk (2.6.12)
 
 PLATFORMS
-  arm64-darwin-22
-  arm64-darwin-23
-  x86_64-linux
```
Comment (mark-pkgflow) [inline Gemfile.lock:182]: We should keep all of these in addition to the newly added `x86_64-darwin-20`

**Review by mark-pkgflow — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (mark-pkgflow) [reply Gemfile.lock:182]: The rest looks spot on! Nice work

**Review by mark-pkgflow — DISMISSED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (AbigailMcP) [reply Gemfile.lock]: Re-added!

**Review by mark-pkgflow — DISMISSED** [review]

**Review by mark-pkgflow — APPROVED** [review]

**Comment by claire-hughez** [comment]
Good work! This all looks right to me. 
The photos worked in the first PR but are all dead now - is that just something that will be added later?

**Comment by AbigailMcP** [comment]
> Good work! This all looks right to me. The photos worked in the first PR but are all dead now - is that just something that will be added later?

@claire-hughez this might be to do with the fact that this was the PR i used to test out the deploy previews - potentially just a caching issue since i deployed it a few times testing out settings. I'll keep an eye out on future deploy previews.

**Comment by AbigailMcP** [comment]
Ah i take it back - it looks like it's a problem in production, rather than just this branch.. @claire-hughez

**Comment by claire-hughez** [comment]
@AbigailMcP @mark-pkgflow  do we need a ticket to fix it?

**Pull request merged**